app/Resources/views/autoEdits/result.html.twig

```diff
@@ -0,0 +1,68 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+<div class="col-lg-12">
+<div id="wrapper">
+<main id="content" class="clearfix">
+        <div class="col-lg-8 col-lg-offset-2">
+            </div>
+    
+    <div class="panel panel-primary">
+    <header class="panel-heading">
+        <div class="text-center xt-heading-top">
+            <a class="back-to-search" href="{{ path("autoedits") }}"">
+                <span class="glyphicon glyphicon-chevron-left"></span>
+                Back
+            </a>
+            {{ wiki_link("User:" ~ username, url, username) }} &middot; {{ wikiName }}
+        </div>
+    </header>
+    <div class="panel-body xt-panel-body">
+    		<p>
+				<a href="{{ linkWikiScript(url, "title=Special:Log&type=block&user=&year=&month=-1&tagfilter=&user=User:"~username) }} ">{{ msg("block_log") }}</a>
+				{% if isWMFLabs() %}
+				&middot;
+				<a href="//tools.wmflabs.org/guc/?user={{username}}" >Global user contributions</a>
+				&middot;
+				<a href="//meta.wikimedia.org/w/index.php?title=Special%3ACentralAuth&target={{username}}" >Global Account Manager</a>
```

Comment:
I don't think we have an i18n message for this, but I would just call it "SUL" or "CentralAuth". We can worry about that later.

Also, you might consider testing for `app.single_wiki` and not `isWMFLabs` here. You should be able to do:
`{{ wiki_link('Special:CentralAuth/'~username, url) }}` since that Special page will work on any wiki